content/en/docs/concepts/virtualization.md

```diff
@@ -1,68 +1,86 @@
+Unikernels come somewhere between those 2.
 
-### OS-Level Virtualization and Containers
+### Virtual Machines
 
-Unikernels are not containers and are not based on container technology.
-Instead, unikernels are a replacement for containers entirely
+A virtual machine represents an abstraction of the hardware, over which an operating system can run, thinking that it is alone on the system and that it controls the hardware below it.
+Virtual machines rely on hypervisors to run properly.
 
-### Hyerpvisors and Virtual Machines
+A hypervisor incorporates hardware-enabled multiplexing and segmentation of compute resources in order to better utilize, better secure and better facilitate the instantenous runtime of user-defined programs. Through By the means of a virtaul machine, an operating system is unaware of the multiplexing which happens to facilitate its existence. The hypervisor emulates devices on behalf of the guest OS, including providing access to virtual CPUs, virtual Interrupt Controllers and virtual NICs, which are segmented from the underlying hardware.
 
-A hypervisor incorporates hardware-enabled multiplexing and segmentation of
-compute resources in order to better utilize, better secure and better
-facilitate the instantenous runtime of user-defined programs.  Traditionally,
-monolithic Operating Systems (OSes) can become one or many Virtual Machines
-(VMs), acting as if they the sole utilizer of their resources, unaware of the
-multiplexing which happens to facilitate its existence.  Unikraft can be used to
-build bespoke application-specific VMs which can run alongside monolithic OS
-VMs.  The figure below illustrates an enumeration of at least one VM supervised
-by a hypervisor embedded as a kernel component toolkit in the first OS.
+The hypervisors can be classified in 2 categories: Type 1 and Type 2:
 
+* The **Type 1 hypervisor**, also known as **bare-metal hypervisor**, has direct access to the hardware and controls all the operating systems that are running on the system.
+  The guest OSes have access to the physical hardware, and the hypervisor arbiters this accesses.
+  KVM is an example of Type 1 hypervisor.
+* The **Type 2 hypervisor**, also known as **hosted hypervisor**, has to go through the host operating system to reach the hardware.
+  Access to the hardware is emulated, using software components that behave in the same way as the hardware ones.
+  An example of Type 2 hypervisor is VirtualBox.
 
+| ![type 1 hypervisor os](/assets/imgs/vm1.svg) | ![type 2 hypervisor os](/assets/imgs/vm2.svg) |
+| :--:									  | :--:								    |
+| Operating systems over type 1 hypervisor				  | Operating systems over type 2 hypervisor 				    |
 
-The hypervisor emulates devices on behalf of the guest OS, including providing
-access to vCPUs, vRAM, VFS' and vNIC(s) which are segmented from the underlying
-hardware.
+### Containers
```

Comment:
Does it make sense to discuss containers? I know it's useful information, but it's not directly tied to Unikraft.